website_and_docs/content/documentation/grid/getting_started.en.md

```diff
@@ -47,7 +47,7 @@ single machine.
 **Standalone** is also the easiest mode to spin up a Selenium Grid. By default, the server 
 will listen for `RemoteWebDriver` requests on [http://localhost:4444](http://localhost:4444). 
 By default, the server will detect the available drivers that it can use from the System 
-[`PATH`]({{< ref "../webdriver/getting_started/install_drivers.md#3-the-path-environment-variable" >}}).
+[`PATH`]({{< ref "../webdriver/troubleshooting/errors/driver_location.md#3-the-path-environment-variable" >}}).
```

Comment:
I think this link should have been `driver_location/#use-the-path-environment-variable`

Reply:
Not sure, I think it is better to give the whole overview and let the use choose.